content/blog/2021/containerqueries.md

```diff
@@ -0,0 +1,162 @@
+
+### Apply @-rules at desired breakpoints
+Now that a containment context has been defined, the `@container` rule is used to tell the browser when and how styles should change inside each container. 
+
+With a narrow parent container, the `blockquote` features the image stacked on top of the text. 
+
+{{ embed.figure(
+  data=[{'img': 'blog/2021/narrowquote.png',
+         alt: 'profile image on top of quote'}],
+  caption='The profile image sits on top of the quote in narrow containers.'
+) }}
+
+At the first breakpoint, the layout for the `blockquote` changes so that the profile image moves from being above the quote to sitting next to it, and the text describing the speaker gets a heavier weight applied. 
+
+{{ embed.figure(
+  data=[{'img': 'blog/2021/break1.jpg',
+         alt: 'profile image next to quote'}],
+  caption='The profile image changes position at the first breakpoint to sit next to the quote.'
+) }}
+
+```scss
+/* change styles according to container size */
+@container (min-width: 35em) {
+
+  .quote {
+    grid-gap: 1rem;
+    grid-template: "media quote" auto/ max-content 1fr;
+  }
+
+  .media {
+    align-items: center;
+  }
+
+  .source {
+    font-weight: bold;
+  }
+}
+```
+
+At the second breakpoint, the text for the quote and the attribution gets bigger, the background changes color, and the overall shape of the `blockquote` changes by way of a `clip-path`. 
+
+{{ embed.figure(
+  data=[{'img': 'blog/2021/break2.jpg',
+          alt: 'blockquote with magenta background and clip-path'}],
+  caption='At a wider breakpoint, the blockquote gets a 
+  clip-path and a different background color.'
+) }}
+
+```scss
+@container (min-width: 60em) {
+
+  .quote {
+    --quote-size: 1.5rem;
+    --quote-background: darkmagenta;
+    clip-path: polygon(
+      -0.04% 100.6%,
+      1.12% -0.52%,
+      95.65% 11.18%,
+      96.46% 90.05%
+    );
+    padding: 1.25em 3em;
+    max-width: unset;
+  }
+
+  .source {
+    font-size: 1.1rem;
+  }
+}
+```
+
+Any styles inside of one of these `@container` blocks will be applied when the container fulfills the condition of the query. 
+
+Just as with `@media` queries, authors can write as many `@container` queries as needed.
+
+## All together now
+
+With each `blockquote` sitting in a different sized container, they all look slightly different, at the same viewport size. 
+
+{{ embed.figure(
+  data=[{'img': 'blog/2021/allquotes.png',
+          alt: 'three blockquotes, styled differently'}],
+  caption='The same blockquote component gets styled differently depending on its container size.'
+) }}
+
+
+## What's next?
+
+It's still very early in the proposal process, so a lot regarding how container queries work could change. 
+
+There might be a way to contain only the block axis. Queries could also be made against properties like `aspect-ratio` or `orientation.`
```

Comment:
This might read nicely as a list?

```suggestion
- There might be a way to contain only the block axis. 
- Queries could also be made against properties like `aspect-ratio`, `orientation`, or even [custom properties and layout states](https://github.com/w3c/csswg-drafts/issues/5989).
```

Another interesting one you could add to that list is that we'll likely get [container-based units](https://github.com/w3c/csswg-drafts/issues/5888)

Reply:
added a reference to this